README.md

```diff
@@ -137,6 +137,10 @@ While *build-and-inspect-python-package* will build a wheel for you by default,
   Requires `attestations: write` and `id-token: write` permissions.
   The only meaningful value is `'true'` (note the quotes – GitHub Actions only allow string inputs) and everything else is treated as falsey.
   (*optional*, default: `'false'`).
+- `package`: The package to build.
+  If not specified, the package in the current directory is built.
+  If set to `all`, all packages in the `uv` [workspace](https://docs.astral.sh/uv/concepts/workspaces/) are built.
```

Comment:
Wdyt of adding a note that one can use `uv tree --depth 0` to view all packages in the current workspace?